cassandane/Cassandane/Cyrus/ID.pm

```diff
@@ -69,6 +69,11 @@ sub test_cmd_id
 {
     my ($self) = @_;
 
+    if ($self->{instance}->{have_syslog_replacement}) {
+        # Purge any syslog lines before this test runs.
+        $self->{instance}->getsyslog();
+    }
```

Comment:
FYI This part doesn't need to be wrapped in the have_syslog_replacement check, since we don't care about the logged lines, so it doesn't matter if there weren't any due to a missing syslog replacement :)

Reply:
Legit, I amended things.